report-summary-merged-prs

```diff
@@ -1636,6 +1643,8 @@ if __name__ == "__main__":
     for release_queue_results in results:
         find_dup_dict_result(release_queue_results['comparisons'])
 
+    find_ubsan_logs(release_queue_results['comparisons'], 'noarch')
```

Comment:
this should be under the `for` loop above. We want to run it for all releases (`results` is list of all releases).  May be run if only if `release_queue_results['release_name'] ` contains "UBSAN".

I will suggest to also add the architecture in the result path. This will allow us to run UBSAN for multiple archs e.g. one day when we want to test gcc11 UBSAN then we can easily do it